pkg/daemon/ceph/osd/volume_test.go

```diff
@@ -832,6 +842,160 @@ func TestInitializeBlock(t *testing.T) {
 		}
 	}
 
+	// Test with metadata partition devices specified in the global OSD configuration or in the node level OSD configuration
```

Comment:
Could you also check "two devices specifies the same partition" case as shown below?

```
devices:
- name: sdb
  config:
    metadataDevice: sdd1
- name: sdc
  config:
    metadataDevice: sdd1
```

It corresponds to L761-L764 code block.